GDJS/Runtime/howler-sound-manager/howler-sound-manager.js

```diff
@@ -268,11 +270,12 @@ gdjs.HowlerSoundManager.prototype.getMusicOnChannel = function(channel) {
 };
 
 gdjs.HowlerSoundManager.prototype.setGlobalVolume = function(volume) {
-	Howler.volume(volume/100);
+	this._globalVolume = volume;
```

Comment:
I think here we should ensure that this._globalValue is not > 100 or < 0:

```
if (this._globalVolume > 100) this._globalVolume = 100;
if (this._globalVolume < 0) this._globalVolume = 0;
```

Otherwise getGlobalVolume could return values that are outside [0, 100] :) 
I think Howler.js is discarding the value if it's not between 0 and 1.